python/README.md

```diff
@@ -1,25 +1,13 @@
-# Configuration Script
-This script helps you configure your AWS credentials into the requiered files and initialize terraform backends.
-
-To run this script you need to configure first a virtual environment:
-
-```
-cd terraform-academy/python
-pip3 install virtualenv
-python3 -m venv .
-source bin/activate
-pip3 install -r requirements.txt
-```
-
-After this, you need to create a `variables.ini` file, following the same format as the `variables.ini.example`. Here, you'll add your AWS credentials, profile name and environment.
-
-To configure your AWS credentials run:
-`python3 awsconfig.py`
-
-After running this command, you need to run the following command:
-`source ./source.sh`
-
-To initialize a Terraform backend on a specific lesson run:
+# Configuration Steps
+These steps helps you configure your AWS credentials into the requiered files and initialize terraform backends.
+
+1. cd root of your git repo
```

Comment:
Should we change this to `1. Change directory` so we can comply with the style on the rest of the README?